### PR TITLE
Audiences feature

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Will pick the TOC restructuring changes after this release when we move to the new Hugo website

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by AchuthaSourabhC — APPROVED** [review]

File: docs/sources/reverse-etl/features/audiences.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: "Audiences"
+description: Build audiences on the warehouse sources and activate them in the downstream destinations.
+---
+
+**Audiences** are customer lists that satisfy a specific set of criteria out of all the users of a product. RudderStack's **Audiences** feature lets you create these target customer lists with easy-to-use filters on your warehouse tables. Once created, you can activate them in any RudderStack-supported destination connected to the warehouse. 
+
+For example, you can create audiences and send that data to the downstream digital advertising destinations to:
+
+- Retarget all users from California in order to improve repeat sales.
```
Comment (ameypv-rudder) [inline docs/sources/reverse-etl/features/audiences.mdx:10]:
```suggestion
- Retarget all users from California to improve repeat sales.
```

File: docs/sources/reverse-etl/features/audiences.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+<GhBadge
+  url={'https://rudderstack.com/enterprise-quote'}
+  label={'Plan'}
+  message={'Enterprise'}
+  color={'blueviolet'}
+/>
+
+## Key features
+
+RudderStack provides much value by providing the audience-building capabilities in its Reverse ETL pipeline where you can:
```
Comment (ameypv-rudder) [inline docs/sources/reverse-etl/features/audiences.mdx:23]:
```suggestion
RudderStack provides much value by providing audience-building capabilities in its Reverse ETL pipeline where you can: 
```

File: docs/sources/reverse-etl/features/audiences.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  url={'https://rudderstack.com/enterprise-quote'}
+  label={'Plan'}
+  message={'Enterprise'}
+  color={'blueviolet'}
+/>
+
+## Key features
+
+RudderStack provides much value by providing the audience-building capabilities in its Reverse ETL pipeline where you can: 
+- Create and sync single audience to multiple downstream systems which is very time consuming to do manually.
```
Comment (ameypv-rudder) [inline docs/sources/reverse-etl/features/audiences.mdx:24]:
```suggestion
- Create and sync a single audience to multiple downstream systems, which is very time-consuming to do manually.
```

**Comment by ishachopra0401** [comment]
Migrated to hugo repo

**Pull request closed without merging**